thoth/storages/graph/postgres.py

```diff
@@ -6786,3 +6809,15 @@ def get_python_package_version_all(self, analysis_document_id: str) -> List[Dict
             )
 
             return [{"package_name": i[1], "package_version": i[2], "location": i[0]} for i in query.all()]
+
+    def get_python_package_version_import_packages_all(self, import_name: str) -> List[str]:
+        """Retrieve Python package name for the given import package name."""
+        with self._session_scope() as session:
+            query = (
+                session.query(PythonPackageVersion)
+                .join(FoundImportPackage)
+                .join(ImportPackage)
+                .filter(ImportPackage.import_package_name == import_name)
+                .filter(PythonPackageVersion.id == FoundImportPackage.python_package_version_id)
+            )
+            return [i.package_name for i in query.all()]
```

Comment:
Can the query return `package_name`, `package_version`, `index_url` triplet so the package is uniquely identified?